examples/simple/client/BUILD.bazel

```diff
@@ -19,6 +19,6 @@ http_server(
         ":client",
     ],
     visibility = [
-        "//:server_group",
+        "//examples/simple:server_group",
```

Comment:
can we... make this the default? As in, bring this into the macro? Or is there a reason why users should be concerned with this? I did it for the example but feels like something that would be easy to obfuscate. Thoughts?

Reply:
definitely should be obfuscated in the macro once we have the client fully pulled in here, but wanted to give the ability for building blocks to explicitly declare visibility if needed.